examples/dynamo_integration/react_benchmark_agent/src/react_benchmark_agent/evaluators/action_completion_evaluator.py

```
@@ -0,0 +1,246 @@
+
+User Goal: {goal}
+
+Agent Response: {response}
+
+Was the user goal addressed in the agent's response? Respond with only "YES" or "NO".
+"""
+
+        try:
+            result = await llm.ainvoke(prompt)
+            result_text = str(result).strip().upper()
+            return "YES" in result_text
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)